nixos/release.nix

```diff
@@ -13,6 +13,8 @@ let
 
   forAllSystems = genAttrs supportedSystems;
 
+  forSubsetOfSystems = systems: genAttrs (intersectLists supportedSystems systems);
```

Comment:
Should this go in release-lib.nix?

Reply:
Uff... There's apparently already an equivalent called `forAllSupportedSystems` in release-lib.nix, but nixos/release.nix is not importing release-lib.nix

Reply:
I'd a) import it, b) move `forAllSystems` to release-lib c) use `forAllSupportedSystems`.

Reply:
Yeah that seems the best (and `forAllSystems` could be implemented in terms of `forAllSupportedSystems` to reduce duplication). 

I hate the name `forAllSupportedSystems` though... given that we have `forAllSystems` which does something for every system passed in `supportedSystems`... and then `forAllSupportedSystems` which does something else.

Reply:
I have no problem with you renaming them. Maybe something like `forAllSystems` and `forTheseSystems`?